### ISSUE TITLE
Samsung Galaxy Tab S4: Boots up to Download Mode after updating to latest Android 14 

### ISSUE THREAD

**Issue by techie6** [phhusson/treble_experimentations#2663]
### Current Behavior:
I have been running Android 14 v2024.02.27 ([aosp-arm64-ab-gapps-14.0-20240227.img.xz](https://github.com/ponces/treble_aosp/releases/tag/v2024.02.27)) from Ponces on my Galaxy Tab S4 for the past few months without any issues.

The tablet is running on kernel 4.4.205

![Device Info](https://github.com/user-attachments/assets/2fd7dc7e-c17c-43e4-96a8-4a9cc8c45ba9)

### Describe the bug
Today I tried updating the tablet to the latest stable Android 14 image ([v2024.08.16](https://github.com/ponces/treble_aosp/releases/tag/v2024.08.16)) by flashing the image to the system partition using TWRP. The flashing process completes successfully, but after rebooting the device, the green Download Mode screen appears instead of actually booting into the system. I tried installing a previous ROM version from May 2024 and wiping the Cache/Dalvik but I also encountered the same issue. I eventually reverted back to v2024.02.27 which made the device boot up normally again like it used to.

Could the new updated ROMs have broken support for older kernels? or am I missing something here?

Thank you for your support.

**Comment by Knoxyknoxy** [comment]
Hi I've just bought the tab S4 and interested in this topic. It's the above then the most stable rom?

**Comment by Knoxyknoxy** [comment]
and is this installed using odin3 as i have only one image file here i thought you need 4

**Comment by techie6** [comment]
> Hi I've just bought the tab S4 and interested in this topic. It's the above then the most stable rom?

It's the only image based on Android 14 that worked for me at that point. However, there are newer images now based on Android 15 which I haven't tested. I would suggest that you gave the latest image a try since it has fixed boot on some devices. Let me know if it works so that I can update mine too.
https://github.com/ponces/treble_aosp/releases/tag/v2024.11.08



> and is this installed using odin3 as i have only one image file here i thought you need 4

You need to install TWRP first using odin, then use TWRP to backup your stock rom and install new rom along with DM-Verity Disabler. Follow the instructions here:
https://xdaforums.com/t/tutorial-how-to-install-treble-roms-aosp-lineage-pixel-experience.4147083/

**Comment by Knoxyknoxy** [comment]
thanks techie yes i reread your comment and got the twrp but its new ground for me so these steps will help when i get the working tab enroute in the next few days. i actually have a tab s4 here i forgot about but remembered when i bought the new one thats been rebooting after the sam screen so been at fixing it past day. loaded stock rom using odin and all went good but still it reboots :( cant see much else to do with that one. repart and erased as well still nothing. was tempted to next try this image

**Comment by Knoxyknoxy** [comment]
oh android 15 yes will have ago

**Comment by Knoxyknoxy** [comment]
ive not had to deal with disable dm-verity i usually fix customers devices using stock rom on odin. look forward to getting my working one and im sure all this will become clear. is there anything you can think of to do with the rebooting one

**Comment by techie6** [comment]
> ive not had to deal with disable dm-verity i usually fix customers devices using stock rom on odin. look forward to getting my working one and im sure all this will become clear. is there anything you can think of to do with the rebooting one

Disabling Dm-verity is only needed for 3rd party roms . It's not needed for a stock rom.

For your rebooting tablet, you can either try downloading the correct latest official firmware and flashing all its components using odin.
https://samfw.com/firmware/SM-T830
https://www.sammobile.com/samsung/galaxy-tab-s4/firmware/#SM-T830

Otherwise, try an alternative AOSP rom, like the one mentioned in this thread.

**Comment by Knoxyknoxy** [comment]
Yes was trying to do a stock rom its a dead, memory issue i think. You install the lastest rom and it still wont load past the samsung screen

New working one has arrived today so will work with it

________________________________

**Comment by Knoxyknoxy** [comment]
no 15 just goes into download mode

**Comment by techie6** [comment]
> i got your working 14 one on the working tablet, woohoo. hate to see the other one go to waste. when trying the rebooting one i get blocked by frp lock and i cant load it up. is there anyway round this? ive tried the stock flash and it just reboots. was hoping the custom might work if i can get past frp lock

Glad to know. For the rebooting tablet, you need to follow the referenced instructions above. Basically you need first to unlock OEM , install TWRP, delete the system/data partition, install the rom using TWRP, and finally install the patched boot image to the boot partition which will disable encryption and any other locks. 


> no 15 just goes into download mode

That's unfortunate. I don't know how else I can report this since no one seems to have been looking into this Github issue for quite sometime. Let me know if you are aware of any other way to report this to the rom developer.

**Comment by Knoxyknoxy** [comment]
problem is i cant get the tablet to load up to unlock the bootloader. the stock rom installs but its still rebooting

no im not sure how to report this but its new so hopefully sorted soon

**Comment by techie6** [comment]
> problem is i cant get the tablet to load up to unlock the bootloader. the stock rom installs but its still rebooting

If the device is not bricked, it might be that you are installing the wrong stock rom for your device. There are normally multiple ones for the same device depending on its model SM-T830 (Wi-Fi only); SM-T835 (LTE), SM-T837A (AT&T) and the CSC (Country).

You may also try flashing the older stock Android 8.1 ROM, see if that makes any change.

If all that doesn't work, try seeking help on Reddit or XDA Forums:
https://www.reddit.com/r/GalaxyTab/
https://xdaforums.com/f/samsung-galaxy-tab-s4-questions-answers.8000/

**Comment by Knoxyknoxy** [comment]
ill try the first one found out its a 835 this broken one. the android 14 you are using really slowed down my 830. i play stfc and it was unplayable. in 10 its great

**Comment by Ihcramed** [comment]
> ### Current Behavior:
> 
> I have been running Android 14 v2024.02.27 ([aosp-arm64-ab-gapps-14.0-20240227.img.xz](https://github.com/ponces/treble_aosp/releases/tag/v2024.02.27)) from Ponces on my Galaxy Tab S4 for the past few months without any issues.
> 
> The tablet is running on kernel 4.4.205
> 
> ![Device Info](https://private-user-images.githubusercontent.com/162782393/378797084-2fd7dc7e-c17c-43e4-96a8-4a9cc8c45ba9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU5OTE5NzQsIm5iZiI6MTczNTk5MTY3NCwicGF0aCI6Ii8xNjI3ODIzOTMvMzc4Nzk3MDg0LTJmZDdkYzdlLWMxN2MtNDNlNC05NmE4LTRhOWNjOGM0NWJhOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDEwNFQxMTU0MzRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iYmFmYzFjY2NmN2QwMGVjY2QxNTA2YzM1ZTNjNzgyYmVjYTRkZWY3ZDM4YTE4MmFmZGFlOTE3YTk2MDljZTM3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.7AN2sUJ0-DzJd8qgFlhFQrxBrM0E9wsJCQfvf-LY3Ms)
> ### Describe the bug
> 
> Today I tried updating the tablet to the latest stable Android 14 image ([v2024.08.16](https://github.com/ponces/treble_aosp/releases/tag/v2024.08.16)) by flashing the image to the system partition using TWRP. The flashing process completes successfully, but after rebooting the device, the green Download Mode screen appears instead of actually booting into the system. I tried installing a previous ROM version from May 2024 and wiping the Cache/Dalvik but I also encountered the same issue. I eventually reverted back to v2024.02.27 which made the device boot up normally again like it used to.
> 
> Could the new updated ROMs have broken support for older kernels? or am I missing something here?
> 
> Thank you for your support.

Hello,

I can't do an update other than Andy Yan's lineage-20.0-20240516-UNOFFICIAL-arm64_bgN.img.xz because I have the same error: green Download Mode screen

Have you found a solution for the A14 update or for the A15 installation?

Thanks in advance

**Comment by Knoxyknoxy** [comment]
i got a14 on far to slow on many things, 10 back in and its all good

**Comment by Knoxyknoxy** [comment]
hw damage on other one t835

**Comment by techie6** [comment]
> I can't do an update other than Andy Yan's lineage-20.0-20240516-UNOFFICIAL-arm64_bgN.img.xz because I have the same error: green Download Mode screen
> 
> Have you found a solution for the A14 update or for the A15 installation?

Not yet unfortunately. Try reporting the issue in the developer's telegram group: https://t.me/phhtreble

**Comment by phattychops** [comment]
What recovery are you guys using? I've installed the latest TWRP (3.7.0_9-6_mentalmuso) and I'm unable to flash anything above Android 10.  If I install any img file above 10, TWRP can no longer mount the system_root partition.

**Comment by Ihcramed** [comment]
> What recovery are you guys using? I've installed the latest TWRP (3.7.0_9-6_mentalmuso) and I'm unable to flash anything above Android 10. If I install any img file above 10, TWRP can no longer mount the system_root partition.

**Stock rom:**
Country Switzerlan (AUT)
Model 	SM-T830
Product Code 	AUT
Security Patch 	2022-06-01
OS Version 	Android 10
PDA 	T830XXU5CVG2
CSC 	T830OXM5CVG2
Build Date 	2022-07-25
**TWRP:**
3.7.0_9-6_mentalmuso
**Installation:**
- in twrp format data & format all
- install lineage-20.0-20240516-UNOFFICIAL-arm64_bgN.img
- reboot recovery
- install Disable_Dm-Verity_ForceEncrypt_11.02.2020.zip
- install magisk
- reboot system

Pour moi c'est ok comme cela
